test/nesting.js

```diff
@@ -0,0 +1,100 @@
+var assert = require('assert');
+var rewire = require('rewire');
+var postcss = require('postcss');
+var validateSelectors = rewire('../lib/validate-selectors');
+
+describe('getSelectors function', function() {
```

Comment:
Done in https://github.com/postcss/postcss-bem-linter/pull/85/commits/199ceab8a1e528d1b7b35c217272d06914c2ebe9